etc/launch-tae-logservice/launch.toml

```diff
@@ -0,0 +1,13 @@
+logservices = [
+    "./etc/launch-tae-logservice/log1.toml",
+    "./etc/launch-tae-logservice/log2.toml",
+    "./etc/launch-tae-logservice/log3.toml"
+]
+
+dnservices = [
```

Comment:
The "tae" CN does not need a DN, this section could be deleted

Reply:
Ok, I'll remove this section and dn1.toml